static/scripts/topicEdit.js

```diff
@@ -343,41 +343,47 @@ class TopicBlockList extends React.Component {
 						<button
 							type="button"
 							className="btn btn-secondary"
+							data-testid="topic-addcontent-text-btn"
```

Comment:
```suggestion
							data-testid="topic-add-content-text-btn"
```
Hätte ich erwartet aber scheinbar ins euer Schema für das Naming ein anderes.

Reply:
Im Bereich topic sind wir beim Abschnitt add content und darin gibt es unterschiedliche Arten von Content (text, etherpad), um diese Struktur zu verdeutlichen, habe ich addcontent zusammengeschrieben. 
Das war bei anderen IDs auch so, aber wir sollten uns das Naming Schema sicherlich nochmal zur Brust nehmen in der QA.